Gemfile.lock

```diff
@@ -396,4 +396,4 @@ DEPENDENCIES
   xcpretty-travis-formatter
 
 BUNDLED WITH
-   2.3.22
+   2.3.23
```

Comment:
nitpick: Does CI pass without this change?

Reply:
Unfortunately, no. https://github.com/wordpress-mobile/WordPress-iOS/pull/19409